.gitignore

```diff
@@ -22,3 +22,5 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+
```

Comment:
Please add yarn and packages lock also. There ale lot of conflicts in ples repo with these files. Npm i is still run via pipeline